codecparsers/h265Parser.cpp

```diff
@@ -180,9 +209,9 @@ bool NalUnit::parseNaluHeader(const uint8_t* data, size_t size)
     // forbidden_zero_bit
     br.skip(1);
 
-    nal_unit_type = br.read(6);
-    nuh_layer_id = br.read(6);
-    nuh_temporal_id_plus1 = br.read(3);
+    READ_BITS(br, nal_unit_type, 6);
```

Comment:
@xuguangxin  like what i did in the macro definition, i think it is a better way to let the readers understand our code easier. They will know that we read from br or nr otherwise, they may be confused.